metricshub-engine/src/main/java/org/sentrysoftware/metricshub/engine/connector/parser/ConnectorLibraryParser.java

```diff
@@ -82,8 +82,14 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 			}
 			final ConnectorParser connectorParser = ConnectorParser.withNodeProcessorAndUpdateChain(file.getParent());
 
-			final Connector connector = connectorParser.parse(file.toFile());
-			connectorsMap.put(filename.substring(0, filename.lastIndexOf('.')), connector);
+			try {
+				final Connector connector = connectorParser.parse(file.toFile());
+				connectorsMap.put(filename.substring(0, filename.lastIndexOf('.')), connector);
+			} catch (Exception e) {
```

Comment:
We need to manage the exceptions as well in the `readZipFile` method.

Reply:
Also, would it be possible to get the location of the error? (at least the line number?)

Reply:
When the yaml structure is wrong, we do get the line numbers:
![image](https://github.com/sentrysoftware/metricshub/assets/64595296/8d0834e9-1743-45a5-a9fb-06c8e7b744c4)

However, for cases like wrong computes like unaccepted types (type: substract12345) we get the full context of the error. (Monitor, Source, etc.) We probably need to struggle a bit with jackson to pass the exact line number and column to the context parser.

See with @NassimBtk, this can be done at a further time in a separate issue.

`[ERROR] [o.s.m.e.c.p.ConnectorLibraryParser] Error while parsing connector Windows.yaml: Could not resolve type id 'excludeMatchingLines12345' as a subtype of `org.sentrysoftware.metricshub.engine.connector.model.monitor.task.source.compute.Compute`: known type ids = [add, and, append, arraytranslate, awk, convert, divide, duplicatecolumn, excludematchinglines, extract, extractpropertyfromwbempath, json2csv, keepcolumns, keeponlymatchinglines, leftconcat, multiply, perbittranslation, prepend, replace, rightconcat, substring, subtract, translate, xml2csv] (for POJO property 'computes')
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: org.sentrysoftware.metricshub.engine.connector.model.Connector["monitors"]->java.util.LinkedHashMap["cpu"]->org.sentrysoftware.metricshub.engine.connector.model.monitor.SimpleMonitorJob["simple"]->org.sentrysoftware.metricshub.engine.connector.model.monitor.task.Simple["sources"]->java.util.LinkedHashMap["processorInformation"]->org.sentrysoftware.metricshub.engine.connector.model.monitor.task.source.WmiSource["computes"]->java.util.ArrayList[0])
Performing detection on localhost using WMI...`